scripts/cmd/tee.bat

```diff
@@ -1,5 +1,18 @@
 @ECHO OFF
 
+:: Copyright 2018 ABSA Group Limited
```

Comment:
I didn't want to do that right away, before discussing with you guys.
This script is from Internet, not written by us, nor for Absa...

Reply:
Oh, I did not know that... so what is the original license there? can we even use it?

Reply:
The contribution is at the bottom. I kept the file as it was. Doesn't have any license mentioned on the site, the other doesn't even exist anymore. I think it's a pretty old script.